adapters/pubmatic.go

```diff
@@ -56,7 +56,12 @@ func (a *PubmaticAdapter) Call(ctx context.Context, req *pbs.PBSRequest, bidder
 		}
 		pbReq.Imp[i].Banner.Format = nil // pubmatic doesn't support
 		pbReq.Imp[i].TagID = params.AdSlot
-		pbReq.Site.Publisher = &openrtb.Publisher{ID: params.PublisherId}
+		if pbReq.Site != nil {
+			pbReq.Site.Publisher = &openrtb.Publisher{ID: params.PublisherId}
+		}
```

Comment:
re: openrtb, a request can be Site or App so we might want to do some validation around this.